### PR TITLE
SPARK-1520

### PR DESCRIPTION
**Pull request by speedy01**
This should get the presences and return system icon to the correct icon.

### CODE REVIEW

File: src/java/org/jivesoftware/sparkimpl/plugin/systray/SysTrayPlugin.java [hunk 1]
```diff
@@ -58,6 +59,7 @@
 import org.jivesoftware.sparkimpl.settings.local.SettingsManager;
 import org.jivesoftware.smack.chat.ChatManagerListener;
 import org.jivesoftware.smackx.chatstates.ChatStateListener;
+import org.lobobrowser.html.domimpl.HTMLElementBuilder;
```
Comment (akrherz) [inline src/java/org/jivesoftware/sparkimpl/plugin/systray/SysTrayPlugin.java]: What's going on with this new import?  Sorry for the ignorance

Comment (wrooot) [inline src/java/org/jivesoftware/sparkimpl/plugin/systray/SysTrayPlugin.java]: yeah, it doesn't look that it is needed there

Comment (speedy01) [inline src/java/org/jivesoftware/sparkimpl/plugin/systray/SysTrayPlugin.java]: Yeah, that import should not be there . Im not sure how that got in there ! Ill remove it!

**Comment by wrooot** [comment]
speedy, sorry, i have botched your PR with a revert i had to do to another PR. Could you please submit new one?

**Pull request merged**